src/devices/cpu/i386/i386.cpp

```diff
@@ -273,7 +273,7 @@ void i386_device::i386_load_segment_descriptor(int segment )
 		{
 			if( !m_performed_intersegment_jump )
 				m_sreg[segment].base |= 0xfff00000;
-			if(m_cpu_version < 0x5000)
+			if(m_cpu_version < 0x500)
 				m_sreg[segment].flags = 0x93;
```

Comment:
What's actually going on here?

While 0x5000 made it always run the code block, what would be special about 0x500 in the proposed fix? This kind of change should come with an explanation of the meaning of the registers being set and of the "magic" numbers being used. Preferable with references to the datasheets, if the behavior is described in official documentation.

Reply:
Nope it shouldn't be reverted, this was an obvious typo from https://github.com/mamedev/mame/commit/649c169c2ffb5daa68925114fce0d6d1159d61c0#diff-3cb160bd796b24b4e23eaef590cd5696R276

0x500 is Pentium.

Reply:
Thanks for the clarification!

Reply:
just a quick thought inspired by this: perhaps having `#define`s for the known CPU IDs could make the semantics of this code clearer.

Reply:
Or enums rather, I am sure :)